ui/app/AppLayouts/Chat/ContactsColumn.qml

```diff
@@ -101,84 +311,22 @@ Rectangle {
         }
     }
 
-    SearchBox {
-        id: searchBox
-        anchors.top: title.bottom
-        anchors.topMargin: Style.current.padding
-        anchors.right: addChat.left
-        anchors.rightMargin: Style.current.padding
-        anchors.left: parent.left
-        anchors.leftMargin: Style.current.padding
-    }
-
-    AddChat {
-        id: addChat
-        anchors.right: parent.right
-        anchors.rightMargin: Style.current.padding
-        anchors.top: title.bottom
-        anchors.topMargin: Style.current.padding
-    }
-
-    Connections {
-        target: profileModel.contacts
-        onContactRequestAdded: {
-            if (!appSettings.notifyOnNewRequests) {
-                return
-            }
-            const isContact = profileModel.contacts.isAdded(address)
-            systemTray.showMessage(isContact ? qsTr("Contact request accepted") :
-                                               qsTr("New contact request"),
-                                   isContact ? qsTr("You can now chat with %1").arg(Utils.removeStatusEns(name)) :
-                                               qsTr("%1 requests to become contacts").arg(Utils.removeStatusEns(name)),
-                                   SystemTrayIcon.NoIcon,
-                                   Constants.notificationPopupTTL)
```

Comment:
I've moved this to `AppMain.qml` because this code doesn't interact with any of the components that live in this file and seems rather unrelated.